docs/FAQ.md

```diff
@@ -36,7 +36,7 @@ more, read our blog post,
 
 Yes, we've already started releasing open source versions of some of the plugins
 we use here, and we'll continue to do so.
-[Plugins](https://github.com/spotify/faq#what-is-a-plugin-in-backstage) are the
+[Plugins](https://github.com/spotify/backstage/faq#what-is-a-plugin-in-backstage) are the
```

Comment:
Same here — link should be:
https://github.com/spotify/backstage/blob/master/docs/FAQ.md#what-is-a-plugin-in-backstage